src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,125 +1,178 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+import java.util.Locale;
 import java.util.Optional;
+import java.util.stream.Collectors;
 
 public class JavaDateTimeApi {
+
+    public static final int DEFAULT_LENGTH_LOCAL_DATE = 10;
+    public static final String DELIMITER = "-";
+    public static final String SHORT_IDS_ECT = "ECT";
+    public static final String PATTERN_DATE_YYYY_MM_DD = "yyyyMMdd";
+    public static final String PATTERN_DATE_DD_MMM_YYY = "dd MMM yyy";
+    public static final String PATTERN_DATE_DD_MMMM_YYYY_HH_MM = "dd MMMM yyyy HH:mm";
```

Comment:
I am pretty sure that some of this pattern already exists in https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/format/DateTimeFormatter.html class

Reply:
done